ajax/libs/post-robot/0.0.1/index.min.js

```diff
@@ -0,0 +1,7 @@
+<html>
+<head><title>301 Moved Permanently</title></head>
+<body bgcolor="white">
+<center><h1>301 Moved Permanently</h1></center>
+<hr><center>nginx/1.4.6 (Ubuntu)</center>
+</body>
+</html>

```

Comment:
But I don't think this is the `index.min.js` real content.
This looks like the HTML file.
How about remove the `index.min.js` and run again?
Thank you.